lib/engine.js

```diff
@@ -146,6 +147,21 @@ Engine.prototype._configureApp = function() {
     contracts: this.contracts,
     redis: this.redis
   });
+  const corsError = new Error('Not allowed by CORS');
+  const corsOptions = {
+    origin: function (origin, next) {
+      const originAllowed = function () {
+        return process.env.NODE_ENV !== 'production' ?
+          true : '^https?://(\w+\.)+storj\.io$'.test(origin);
+      };
```

Comment:
This will set the `access-control-allow-origin` header of the response equal to the `origin` header on the request when not in production, and when in production, where `origin` matches the regex (`^https?://(\w+\.)+storj\.io$`).

Reply:
This should be configurable value, correct?